Automattic/wp-calypso
# Logged-in Performance Profiler: Show recommendations and filter by selected recommendation

Closes https://github.com/Automattic/dotcom-forge/issues/9177

## Proposed Changes

* Display the number of recommended fixes on each metric
* Allow filtering recommendations based on selected metric

![image](https://github.com/user-attachments/assets/d24ff35b-d678-4d6f-be32-846595b8f500)


## Why are these changes being made?


*

## Testing Instructions



* Go to `/site-performance/site`
* Select a metric and click `view recommendations`
* Verify it scroll to recommendations section and it's filtered by the selected metric

## Pre-merge Checklist
